src/components/console/views/ActivityErrors.svelte

```diff
@@ -0,0 +1,181 @@
+<svelte:options immutable={true} />
+
+<script lang="ts">
+  import type { ICellRendererParams, IRowNode } from 'ag-grid-community';
+  import type { DataGridColumnDef } from '../../../types/data-grid';
+  import type { ActivityErrorCategories, ActivityErrorCounts, ActivityErrorRollup } from '../../../types/errors';
+  import ActivityErrorsRollup from '../../ui/ActivityErrorsRollup.svelte';
+  import DataGrid from '../../ui/DataGrid/DataGrid.svelte';
+  import TabPanel from '../../ui/Tabs/TabPanel.svelte';
+  import { ConsoleContextKey } from '../Console.svelte';
+
+  type ActivityErrorsRollupRendererParams = ICellRendererParams<ActivityErrorRollup>;
+
+  export let activityValidationErrorRollups: ActivityErrorRollup[] = [];
+  export let activityValidationErrorTotalRollup: ActivityErrorCounts;
+  export let title: string;
+
+  function doesExternalFilterPass({ data }: IRowNode<ActivityErrorRollup>) {
+    if (data) {
+      switch (selectedCategory) {
+        case 'extra':
+          return data.errorCounts.extra > 0;
+        case 'invalidParameter':
+          return data.errorCounts.invalidParameter > 0;
+        case 'missing':
+          return data.errorCounts.missing > 0;
+        case 'wrongType':
+          return data.errorCounts.wrongType > 0;
+        case 'invalidAnchor':
+          return data.errorCounts.invalidAnchor > 0;
+        case 'outOfBounds':
+          return data.errorCounts.outOfBounds > 0;
+      }
+    }
+    return false;
+  }
+
+  function isExternalFilterPresent() {
+    return selectedCategory !== 'all';
+  }
+
+  function onSelectCategory(event: CustomEvent<ActivityErrorCategories>) {
+    const { detail: value } = event;
+    if (value != null) {
+      selectedCategory = value;
+    }
+
+    dataGrid.onFilterChanged();
+  }
+
+  const columnDefs: DataGridColumnDef<ActivityErrorRollup>[] = [
+    {
+      field: 'type',
+      filter: 'string',
+      headerName: 'Activity',
+      resizable: true,
+      sortable: true,
+      width: 60,
+    },
+    {
+      field: 'id',
+      filter: 'number',
+      headerName: 'ID',
+      resizable: true,
+      sortable: true,
+      suppressAutoSize: true,
+      suppressSizeToFit: true,
+      width: 60,
+    },
+    {
+      field: 'fields',
+      filter: 'number',
+      headerName: '# fields',
```

Comment:
This column isn't sorting for some reason - would probably be useful if possible? 
<img width="885" alt="image" src="https://github.com/NASA-AMMOS/aerie-ui/assets/4419607/842518e1-4215-405a-8545-7aa11407440f">

Reply:
Oh that shouldn't necessarily be sortable. I forgot to disable it. How should it be sorted if it is sortable?

Reply:
ohh `fields`! Sorry...I was looking at `issue` for some reason...